src/components/scenes/SendScene2.tsx

```diff
@@ -681,24 +681,25 @@ const SendComponent = (props: Props) => {
       return
     }
 
-    // if (guiMakeSpendInfo.fioPendingRequest != null) {
-    // const { fioPendingRequest: pendingRequest } = guiMakeSpendInfo
-    // try {
-    //   await recordSend(fioWallet, fioAddress, {
-    //     fioRequestId: pendingRequest.fio_request_id,
-    //     payeeFioAddress: pendingRequest.payee_fio_address,
-    //     payerPublicAddress: pendingRequest.payer_fio_public_key,
-    //     payeePublicAddress: pendingRequest.content.payee_public_address,
-    //     amount: pendingRequest.content.amount,
-    //     currencyCode: pendingRequest.content.token_code.toUpperCase(),
-    //     chainCode: pendingRequest.content.chain_code.toUpperCase(),
-    //     txid: edgeSignedTransaction.txid,
-    //     memo
-    //   })
-    // } catch (e: any) {
-    //   const message = e?.message ?? ''
-    //   message.includes(FIO_FEE_EXCEEDS_SUPPLIED_MAXIMUM) ? showError(lstrings.fio_fee_exceeds_supplied_maximum_record_obt_data) : showError(e)
-    // }
+    if (fioPendingRequest != null) {
+      try {
+        await recordSend(fioWallet, fioSender.fioAddress, {
+          fioRequestId: fioPendingRequest.fio_request_id,
+          payeeFioAddress: fioPendingRequest.payee_fio_address,
+          payerPublicAddress: fioPendingRequest.payer_fio_public_key,
+          payeePublicAddress: fioPendingRequest.content.payee_public_address,
+          amount: fioPendingRequest.content.amount,
+          currencyCode: fioPendingRequest.content.token_code.toUpperCase(),
+          chainCode: fioPendingRequest.content.chain_code.toUpperCase(),
+          txid,
+          memo: fioSender.memo
+        })
+      } catch (e: any) {
+        const message = e?.message ?? ''
```

Comment:
Do `const message = e instanceof Error ? e.message : ''`

Reply:
ok. Can we just do our usual `String(e)` that we've since standardized on

Reply:
That works too